build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group = 'pro.civitaspo'
-version = '0.1.3'
+version = '0.1.4'
```

Comment:
Could you revert this? Bumping up version is responsible for me.

Reply:
fixed